MVC-App/Controllers/tblRelationsController.cs

```diff
@@ -0,0 +1,193 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Data.Entity;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Net;
+using System.Web;
+using System.Web.Mvc;
+using MVC_App.Domain.Models;
+
+namespace MVC_App.Controllers
+{
+    public class tblRelationsController : Controller
+    {
+        private testEntities db = new testEntities();
+
+        private IEnumerable<RelationModel> relationModels;
+
+        public tblRelationsController()
+        {
+            var relations = from tblRelation in db.tblRelation
+                            join tblRelationAddress in db.tblRelationAddress on tblRelation.Id equals tblRelationAddress.RelationId
+                            where tblRelation.IsDisabled != true
+                            select new RelationModel
+                            {
+                                Id = tblRelation.Id,
+                                RelationAddressId = tblRelationAddress.Id,
+                                Name = tblRelation.Name,
+                                FullName = tblRelation.FullName,
+                                TelephoneNumber = tblRelation.TelephoneNumber,
+                                Email = tblRelation.EMailAddress,
+                                Country = tblRelationAddress.CountryName,
+                                City = tblRelationAddress.City,
+                                Street = tblRelationAddress.Street,
+                                PostalCode = tblRelationAddress.PostalCode,
+                                StreetNumber = tblRelationAddress.Number ?? 0
+                            };
+            relationModels = relations.ToList();
+        }
+
+        // GET: tblRelations
+        public async Task<ActionResult> Index()
+        {
+            return View(relationModels);
+        }
+
+        // GET: tblRelations/Details/5
+        public async Task<ActionResult> Details(Guid? id)
+        {
+            if (id == null)
+            {
+                return new HttpStatusCodeResult(HttpStatusCode.BadRequest);
+            }
+            tblRelation tblRelation = await db.tblRelation.FindAsync(id);
```

Comment:
use var where it is possible
var tblRelation = await db.tblRelation.FindAsync(id);